packages/atomic/src/components/atomic-search-interface/atomic-aria-live.tsx

```diff
@@ -0,0 +1,41 @@
+import {Component, h, Host, Listen, State} from '@stencil/core';
+
+export interface UpdateLiveRegionEventArgs {
+  region: string;
+  message: string;
+}
+
+/**
+ * The `atomic-aria-live` component notifies screen readers of changes in the search interface.
+ * @internal
+ */
+@Component({
+  tag: 'atomic-aria-live',
+  shadow: false,
+})
+export class AtomicAriaLive {
+  @State() public regions: Record<string, string> = {};
+
+  @Listen('atomic/accessibility/updateLiveRegion', {target: 'document'})
+  protected updateMessage({
+    detail: {message, region},
+  }: CustomEvent<UpdateLiveRegionEventArgs>) {
+    this.regions = {...this.regions, [region]: message};
+  }
+
+  public render() {
+    return (
+      <Host style={{position: 'absolute', right: '10000px'}}>
+        {Object.entries(this.regions).map(([region, message]) => (
+          <div
+            role="status"
+            aria-live="polite"
+            id={`atomic-aria-region-${region}`}
```

Comment:
If there are multiple search interfaces in a page, could duplicate ids create an issue? Why is an `id` needed?

Reply:
It looks like using the same `id` in different shadowDOMs is permitted.

The `id` isn't strictly needed (or at-least I haven't tested without it), but I think an identifier makes more sense here than a class name, and it'll be useful for testing.